### PR TITLE
Add CRD to operator-policy message

### PR DESCRIPTION
**Pull request by JustinKuli**
This was added to the compliance message in
https://github.com/open-cluster-management-io/config-policy-controller/pull/222

### CODE REVIEW

**Review by dhaiducek — DISMISSED** [review]

**Review by mprahl — DISMISSED** [review]

**Comment by JustinKuli** [comment]
Hmm, looks like a consistent different issue (just before the thing I thought I fixed)... I'll keep looking.

**Review by yiraeChristineKim — APPROVED** [review]

File: test/integration/compliance_history_test.go [hunk 1]
```diff
@@ -614,6 +615,9 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
 		)
 		Expect(err).ToNot(HaveOccurred())
 
+		By("Verifying that the policy has created the namespace")
+		verifyPolicyOnAllClusters(ctx, policyNS, prereqPolicyName, "Compliant", defaultTimeoutSeconds)
```
Comment (dhaiducek) [inline test/integration/compliance_history_test.go:619]: Well. This seems important. 😆

**Review by dhaiducek — DISMISSED** [review]
Thanks! Feels like those prereq policy checks ought to resolve a fair bit of the flakiness... 🤞

**Comment by JustinKuli** [comment]
/hold

I've got one other minor change to add here, which will double as a consistency check for the tests. So I will push that after this run (hopefully) passes.

File: test/integration/compliance_history_test.go (hunk 1 above)
Comment (mprahl) [reply test/integration/compliance_history_test.go:619]: Do you want to bump the timout to `defaultTimeoutSeconds *2`? Below as well?

Comment (JustinKuli) [reply test/integration/compliance_history_test.go:619]: I expect it to be fine at 30 seconds. The config policy controller just needs to get to the policy once. If we have to use `default * 2` for everything, maybe we should just increase the default :laughing:

**Review by yiraeChristineKim — APPROVED** [review]

**Comment by yiraeChristineKim** [comment]
Oh good this fix will fix my PR CI

**Comment by JustinKuli** [comment]
It was a different looking error, so I'd prefer to handle it in a separate PR:
```
Summarizing 1 Failure:
  [FAIL] GRC: [P1][Sev1][policy-grc] Test install Operator When no OperatorGroup is specified [It] Should verify CSV details [BVT]
  /go/src/github.com/stolostron/governance-policy-framework/test/integration/policy_install_operator_test.go:220
```

**Comment by yiraeChristineKim** [comment]
Cherry pick to 2.10?

**Pull request merged**